src/bugjira/util.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+
+def is_bugzilla_key(key):
+    """returns True if the input key is in the format of a bugzilla bug ID
+
+    :param key: The input key to test
+    :type key: str
+    :raises ValueError: If the key is not of type str
+    :return: True if the input key is in the format of a bugzilla bug ID
+    :rtype: bool
+    """
+    if not isinstance(key, str):
+        raise ValueError(f"Key must be a str: {key}")
+    bz = re.compile("^[0-9]+$")
```

Comment:
so this will only recognize bug id number as a string

from rdopkg parsing common forms of bugzilla id is rhbz#bug_id  or other formats.
This is probably ok to start with just confirming since I had to go look to make sure myself

Reply:
For this first round I just forced keys to be a str. I agree it would be convenient to allow passing in an int for a key (it's unambiguously a BZ in that case, which is nice).